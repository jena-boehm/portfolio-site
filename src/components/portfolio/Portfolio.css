.portfolioContainer {
    display: flex;
    justify-content: space-between;
    align-items: center;
    flex-direction: column;
    height: max-content;
    width: 100%;
    box-sizing: border-box;

    .titleLeft {
        transform: rotate(270deg);
        font-size: 5.5em;
        letter-spacing: 0.1em;
        margin: 0;
        padding: 0;
        height: max-content;
        opacity: 0.4;
        position: fixed;
        left: -2.2em;
        top: 4.1em;
    }

    .portfolioMain {
        display: flex;
        justify-content: center;
        align-items: center;
        flex-direction: column;
        height: 100%;
        width: 90%;
        margin: 2em 2em 2em 10em;
        padding: 0;

        .filterSearch {
            width: 100%;
            display: flex;
            justify-content: center;
            align-items: center;
            flex-direction: column;
        }

        .portfolioGrid {
            width: 100%;
            height: max-content;
            display: flex;
            justify-content: center;
            align-items: center;
            flex-direction: row;
            flex-wrap: wrap;
            margin: 2em 0 2em 0;

            .projectItem {
                margin: 1em;
            }

            .projectName {
                text-align: center;
            }

            .projectFigure {

                .projectImage {
                    box-shadow: rgba(0, 0, 0, 0.4) 0px 50px 100px -20px, 
                    rgba(0, 0, 0, 0.7) 0px 30px 60px -30px, 
                    rgba(0, 0, 0, 0.7) 0px -2px 6px 0px inset;
                    border: 1px solid rgba(255, 255, 255, 0.1);
                }

                .projectImage:hover {
                    box-shadow: rgba(0, 0, 0, 0.7) 0px 50px 100px -20px, 
                    rgba(0, 0, 0, 0.7) 0px 30px 60px -30px, 
                    rgba(0, 0, 0, 0.7) 0px -2px 6px 0px inset;
                    transform: scale(1.05);
                    cursor: pointer;
                }

                .projectImage:active {
                    transform: scale(1)
                }
            }
        }
    }
}