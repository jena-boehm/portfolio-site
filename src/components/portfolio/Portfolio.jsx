import React from 'react';
import styles from './Portfolio.css';

export const Portfolio = () => {
  return (
    <div className={styles.portfolioContainer}>
            Portfolio
    </div>
  );
};
