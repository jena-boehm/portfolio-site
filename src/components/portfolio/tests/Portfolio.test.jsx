import React from 'react';
import { render, cleanup } from '@testing-library/react';
import { Portfolio } from '../Portfolio';

describe('Portfolio component', () => {
  afterEach(() => cleanup());
  it('renders Portfolio', () => {
    const { asFragment } = render(<Portfolio />);
    expect(asFragment()).toMatchSnapshot();
  });
});
