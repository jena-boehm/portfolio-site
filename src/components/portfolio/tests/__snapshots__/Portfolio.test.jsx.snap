// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Portfolio component renders Portfolio 1`] = `
<DocumentFragment>
  <div
    class="portfolioContainer"
  >
    Portfolio
  </div>
</DocumentFragment>
`;
